src/main/java/dev/slimevr/vr/processor/skeleton/LegTweaks.java

```diff
@@ -0,0 +1,822 @@
+			) {
+				rightFootPosition.y = lastPositionY;
+				correctRight = false;
+			}
+		}
+
+		// using the velocity correct the position
+		if (correctLeft) {
+			correctLeftFootTrackerY();
+		}
+
+		if (correctRight) {
+			correctRightFootTrackerY();
+		}
+	}
+
+	private void correctLeftFootTrackerY() {
+		Vector3f temp;
+		Vector3f leftFootDif = leftFootPosition
+			.subtract(bufferHead.getParent().getLeftFootPositionCorrected());
+		if (Math.abs(leftFootDif.y) > 0.005f) {
+			temp = bufferHead.getParent().getLeftFootPositionCorrected();
+			temp = temp
+				.subtract(
+					bufferHead
+						.getParent()
+						.getLeftFootPosition()
+						.subtract(bufferHead.getLeftFootPosition())
+				);
+			float leftFloor = (floorLevel + (MAX_DYNAMIC_DISPLACEMENT * getLeftFootOffset()))
+				- currentDisengagementOffset;
+			if (
+				temp.y
+					< leftFloor
+			) {
+				leftFootPosition.y = leftFloor;
+			} else {
+				leftFootPosition.y = temp.y;
+			}
+
+			Vector3f velocity = bufferHead.getLeftFootVelocity();
+			if (velocity.y * leftFootDif.y > 0) {
+				leftFootPosition.y += velocity.y * FOOT_Y_CORRECTION_WEIGHT;
+			} else {
+				leftFootPosition.y -= velocity.y * FOOT_Y_CORRECTION_WEIGHT;
+			}
+			// check for overshoot and correct if necessary
+			if (
+				checkOverShoot(
+					this.bufferHead.getLeftFootPosition().y,
+					this.bufferHead.getParent().getLeftFootPositionCorrected().y,
+					leftFootPosition.y
+				)
+			) {
+				leftFootPosition.y = bufferHead.getLeftFootPosition().y;
+			}
+		}
+	}
+
+	private void correctRightFootTrackerY() {
+		Vector3f temp;
+		Vector3f rightFootDif = rightFootPosition
+			.subtract(bufferHead.getParent().getRightFootPositionCorrected());
+		if (Math.abs(rightFootDif.y) > 0.005f) {
+			temp = bufferHead.getParent().getRightFootPositionCorrected();
+			temp = temp
+				.subtract(
+					bufferHead
+						.getParent()
+						.getRightFootPosition()
+						.subtract(bufferHead.getRightFootPosition())
+				);
+			float rightFloor = (floorLevel + (MAX_DYNAMIC_DISPLACEMENT * getRightFootOffset()))
+				- currentDisengagementOffset;
+			if (
+				temp.y
+					< rightFloor
+			) {
+				rightFootPosition.y = rightFloor;
+			} else {
+				rightFootPosition.y = temp.y;
+			}
+			Vector3f velocity = bufferHead.getRightFootVelocity();
+			if (velocity.y * rightFootDif.y > 0) {
+				rightFootPosition.y += velocity.y * FOOT_Y_CORRECTION_WEIGHT;
+			} else {
+				rightFootPosition.y -= velocity.y * FOOT_Y_CORRECTION_WEIGHT;
+			}
+			// check for overshoot and correct if necessary
+			if (
+				checkOverShoot(
+					this.bufferHead.getRightFootPosition().y,
+					this.bufferHead.getParent().getRightFootPositionCorrected().y,
+					rightFootPosition.y
+				)
+			) {
+				rightFootPosition.y = bufferHead.getRightFootPosition().y;
+			}
+		}
+	}
+
+	// returns true if it is likly the user is standing
+	public boolean isStanding() {
+		// if the waist is below the verticalcutoff, user is not standing
```

Comment:
Add a space between "vertical" and "cutoff".
```suggestion
		// if the waist is below the vertical cutoff, user is not standing
```